src/main/java/uk/gov/hmcts/reform/idam/client/IdamApi.java

```diff
@@ -17,15 +19,35 @@
 
 @FeignClient(name = "idam-api", url = "${idam.api.url}", configuration = CoreFeignConfiguration.class)
 public interface IdamApi {
-    @RequestMapping(method = RequestMethod.GET, value = "/details")
-    UserDetails retrieveUserDetails(@RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation);
 
-    @RequestMapping(method = RequestMethod.POST, value = "/pin")
+    @RequestMapping(
+        method = RequestMethod.GET,
+        value = "/details"
+    )
+    UserDetails retrieveUserDetails(
+        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation
+    );
+
+    @RequestMapping(
```

Comment:
PostMapping would be simpler if you're going to touch this code,

and GetMapping on the rest on the Gets..

Reply:
👍